src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareDefault(a, b) {
+    const firstItem = a.toString();
+    const secondItem = b.toString();
+
+    if (firstItem > secondItem) {
+      return 1;
+    } else if (firstItem === secondItem) {
```

Comment:
IMHO, else we can't use (from checklist)

Reply:
agree